com.usebottles.bottles.yml

```diff
@@ -261,50 +214,6 @@ modules:
       - type: file
         path: ld.so.conf
```

Comment:
You can drop the `i386-libs` and `compat` modules. Needed files and dirs will be inherited from the baseapp.